biz.aQute.launchpad/src/aQute/launchpad/junit/LaunchpadRunner.java

```diff
@@ -159,7 +159,8 @@ public void doProbeBundle(Launchpad launchpad) {
 
 			@SuppressWarnings("unchecked")
 			Class<ProbeImpl> c = (Class<ProbeImpl>) probeBundle.loadClass(ProbeImpl.class.getName());
-			Probe probe = c.newInstance();
+			Probe probe = c.getDeclaredConstructor()
```

Comment:
why is this better?

Reply:
I am not sure getDeclaredConstructor is right. Better to use getConstructor. This change is because the API now strongly suggests using Constructor.newInstance over Class.newInstance. In fact, starting in Java 9 you get a deprecation warning. I have previously made this change globally https://github.com/bndtools/bnd/commit/8ae6af3bb8d252463d9f6e5cbca46044e15251b4

Reply:
I've changed it  to use the the getConstructor Method.